apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsRefControl.tsx

```diff
@@ -30,9 +30,9 @@ export const jsonFormsRefControlTester: RankedTester = rankWith(
 
 const parseHref = (
   href: string,
-  pageType: Omit<LinkTypes, "email" | "page">,
+  pageType: Omit<LinkTypes, typeof LINK_TYPES.Email | typeof LINK_TYPES.Page>,
```

Comment:
issue: this should be Exclude

```suggestion
  pageType: Exclude<LinkTypes, typeof LINK_TYPES.Email | typeof LINK_TYPES.Page>,
```